src/iperf_api.c

```diff
@@ -90,6 +95,16 @@
 #include <pthread.h>
 #include <sys/sysinfo.h>
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 18, 0)
+    #error Your configuration isn't supporting LSO dor UDP! If you want to use it on\
```

Comment:
thats wrong. this way you disable iperf builds on all linuxes < 4.18.
thats wrong.